packages/editor-ui/src/views/NodeView.vue

```diff
@@ -4449,6 +4449,19 @@ export default defineComponent({
 				});
 			}
 
+			const lastNodeType = this.nodeTypesStore.getNodeType(lastAddedNode.type);
+			const isSubNode = NodeHelpers.isSubNodeType(lastNodeType);
+
+			if (isSubNode && nodes.length > 1) {
+				this.onMoveNode({
+					nodeName: lastAddedNode.name,
+					position: [
+						lastAddedNode.position[0] - NodeViewUtils.NODE_SIZE * 2.5,
```

Comment:
Could you please add a short comment here on why we move the node?

Reply:
Sure, done